client/pages/docs/[id]/enqueue.vue

```diff
@@ -120,9 +156,22 @@ const selectedLabelIds = ref(defaultSelectedLabelIds.value ?? [])
 watch(
   selectedLabelIds,
   async () => {
-    console.log("CHANGE TO SEND TO API", { ids: selectedLabelIds.value ? Array.from(selectedLabelIds.value) : [] })
+    console.log('CHANGE TO SEND TO API', {
```

Comment:
debug

Reply:
That's in `main` so could I deal with it in another PR? I have a `number[]` of label ids that I'd like to set on the document. And regarding that does an API that sets them all exist or is the API more patch-based (add/remove)?